lib/events.js

```diff
@@ -299,6 +299,28 @@ class AdobeIOEvents {
         return handleFetchErrors(response).json();
     }
 
+    /**
+     * Fetches a journal or webhook registration.
+     * @param {String} registrationId the id of the event listener registration to fetch
+     * @returns {Promise}
+     */
+    async getJournal(journal, registrationId, retryOptions=true) {
+        const consumerId = journal.consumerId || this.defaults.consumerId;
+        const applicationId = journal.applicationId || this.defaults.applicationId;
+
+        const url =`${IO_API_HOST[this.env]}/events/organizations/${consumerId}/integrations/${applicationId}/registrations/${registrationId}`;
+        const response = await fetch(url,{
+            method: 'GET',
+            headers: {
+                'x-api-key': this.auth.clientId,
+                'x-ims-org-id': this.auth.orgId,
+                'Authorization': `Bearer ${this.auth.accessToken}`,
+            },
+            retryOptions: retryOptions
+        });
+        return handleFetchErrors(response).json();
```

Comment:
The description says webhook or journal but it's possible to have both associated with an event provider. In addition, there may still be cases where multiple journals are associated with an event provider.

I expect we need functionality to:

* Find the journals associated with an integration
* From that set find the journal that's associated with an event provider

Reply:
That functionality sounds similar to what we have here in [asset-compute-registration](https://git.corp.adobe.com/nui/asset-compute-registration/blob/2175bd8996e0a0713d63646c7315ecf9a8e87fce/lib/registration.js#L306)?

Reply:
the swagger docs says that this call to getRegistration  https://git.corp.adobe.com/pages/adobeio/channel-subscription-management/#!/v1_registration/getRegistration, will return the same response as the createRegistration https://git.corp.adobe.com/pages/adobeio/channel-subscription-management/#!/v1_registration/createRegistration

Given we already have the registrationId, Bob do you think the registerationId could be of a webhook etc.. ?